app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
it should be an instance attribute

Reply:
It's written in the readme: "Also, engineer should have skills attribute - a list of his/her skills, such as "Python" or "JavaScript". skills should be initialised with empty list."  
I understand that skills is a class attribute. Or am I confused?

Reply:
the list of skills is an attribute of every engineer. so every created instance could have his own set of skills

Reply:
e.x. frontend devs could have some optional skills such as Vue.js or React